spec/spec_helper.rb

```diff
@@ -2,6 +2,7 @@
 require "codeclimate-test-reporter"
 require "capybara/rspec"
 require 'sidekiq/testing'
+require 'factory_girl_rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.